### PR TITLE
feat: [FFM-5221]: PHP one click code example

### PR DESCRIPTION
**Pull request by laura-mannus**
### Summary

Updating the PHP code example in the onboarding flow. It should now align with a one-click code example.

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: src/modules/75-cf/strings/strings.en.yaml [hunk 1]
```diff
@@ -1539,27 +1539,27 @@ onboarding:
...
+      - Installed <a href="https://getcomposer.org/">Composer</a>
+      - <a href="https://ngdocs.harness.io/article/1j7pdkqh7j-create-a-feature-flag">Created a Feature Flag on the Harness Platform</a> called 'harnessappdemodarkmode'
+      - <a href="https://ngdocs.harness.io/article/1j7pdkqh7j-create-a-feature-flag#step_3_create_an_sdk_key">Created a server SDK key and made a copy of it</a>
+
+
+      # Install
       The first step is to install the SDK as a dependency in your application using Composer.
 
       ```shell
       composer require harness/ff-server-sdk
```
Comment (knagurski) [inline src/modules/75-cf/strings/strings.en.yaml:1556]: I think this won't actually work. `composer` (by default) will only grab stable versions. You have to tell it what version to grab if you want to get something unstable (like our beta version)
```suggestion
      composer require harness/ff-server-sdk:0.1.0-beta
```

File: src/modules/75-cf/strings/strings.en.yaml [hunk 2]
```diff
@@ -1569,58 +1569,21 @@ onboarding:
       use Harness\Client;
       use OpenAPI\Client\Model\Target;
 
-      $SDK_KEY = getenv("SDK_KEY") ?: "";  // you can put your key in env variable or you can provide in the code
+      $SDK_KEY = "";  // you can put your key in env variable or you can provide in the code
```
Comment (knagurski) [inline src/modules/75-cf/strings/strings.en.yaml:1572]: Let's put the key in here for them like in the JS example

File: src/modules/75-cf/strings/strings.en.yaml [hunk 3]
```diff
@@ -1569,58 +1569,21 @@ onboarding:
...
       $FLAG_NAME = "harnessappdemodarkmode";
 
       $client = new Client($SDK_KEY, new Target(["name" => "harness", "identifier" => "harness"]));
```
Comment (knagurski) [inline src/modules/75-cf/strings/strings.en.yaml:1575]: Just ran this and unfortunately this SDK has been setup to default to using the proxy, which is a huge barrier to adoption. We should definitely look at this again, but the meantime, let's make the demo work out of the box.


```php
<?php

require_once __DIR__ . '/vendor/autoload.php';

use Harness\Client;
use OpenAPI\Client\Model\Target;

$SDK_KEY = '{{apiKey}}';
$FLAG_NAME = 'harnessappdemodarkmode';

$client = new Client(
    $SDK_KEY,
    new Target(['name' => 'harness', 'identifier' => 'harness']),
    [
        'base_url' => 'https://config.ff.harness.io/api/1.0/',
        'events_url' => 'https://events.ff.harness.io/api/1.0'
    ]
);
$result = $client->evaluate($FLAG_NAME, false);

echo "Evaluation value for flag '$FLAG_NAME' with target 'harness': " . json_encode($result);

```

File: src/modules/75-cf/strings/strings.en.yaml (hunk 1 above)
Comment (laura-mannus) [reply src/modules/75-cf/strings/strings.en.yaml:1556]: Interesting! Should I raise a doc ticket as this is where I got it from?

Comment (laura-mannus) [reply src/modules/75-cf/strings/strings.en.yaml:1556]: Have addressed here, thank you

File: src/modules/75-cf/strings/strings.en.yaml (hunk 2 above)
Comment (laura-mannus) [reply src/modules/75-cf/strings/strings.en.yaml:1572]: Sure, have used the same one as in the JS example

File: src/modules/75-cf/strings/strings.en.yaml (hunk 3 above)
Comment (laura-mannus) [reply src/modules/75-cf/strings/strings.en.yaml:1575]: Appreciate you looking into this thank you, have updated

**Review by knagurski — APPROVED** [review]
Looks about right to me

**Review by crystal-fong — APPROVED** [review]

**Pull request merged**